packages/garbo/src/resources/mayam.ts

```diff
@@ -104,7 +103,7 @@ function getBestGreedyCombination(
 function expandCombinationGroup<N extends number>(
   group: Tuple<MayamCalendar.Combination, N>,
 ): [...Tuple<MayamCalendar.Combination, N>, MayamCalendar.Combination][] {
-  const forbiddenSymbols = [...flat(group), ...MayamCalendar.symbolsUsed()];
+  const forbiddenSymbols = [...group.flat(), ...MayamCalendar.symbolsUsed()];
```

Comment:
I think we need to define a depth here, right? should this be `.flat(2)`?

(libram's `flat` behavior differs from js's, I think)

Reply:
Interesting -- libram defaults to a depth of Infinity, while js defaults to a depth of 1.